projects/core/src/personalization/model/personalization-context.model.ts

```diff
@@ -0,0 +1,13 @@
+export interface PersonalizationAction {
+  customization_name: string;
```

Comment:
I am not sure whether all these fields are mandatory, if not, please add `?` to make it optional. If yes, just ignore this comment :) .